app/assets/javascripts/discourse/models/post-stream.js.es6

```diff
@@ -565,6 +565,15 @@ export default RestModel.extend({
     });
   },
 
+  loadNearestPostToDate(date) {
+    const url = `/posts/by_date/${this.get("topic.id")}/${date}`;
```

Comment:
I made this mistake recently but apparently we should avoid underscore in URLs.

Reply:
Yes but we have : "by_number" So it seemed weird to have one by-date and one by_number